src/dict.c

```diff
@@ -180,6 +191,10 @@ int _dictExpand(dict *d, unsigned long size, int* malloc_failed)
     }
 
     /* Prepare a second hash table for incremental rehashing */
+    /* 由 dict.h 源码可以看出，ht_xxx一般为大小为2的数组
+     * 下标为1的元素用于在重哈希时临时存储新的哈希表
+     * 因此这里将exp，已使用大小以及*新哈希表赋值给下标为1的元素
```

Comment:
```suggestion
     * 因此这里将新哈希表的空间扩展指数，新哈希表已使用大小以及新哈希表指针赋值给下标为 1 的元素
```
@enjoy-binbin 不好意思彬彬春节假期还 at 你😂，想问你一下 exp 如果要翻译成中文的话怎么翻译比较好呢？

Reply:
好像是不太好翻译，原本字典的 size 是存储着大小，然后 7.0 里存储的是对应的指数
比如说之前的 size 是严格的 2 的幂，例如 1024。然后现在 size_exp 存储的是 2 的那个幂，例子里存储的就是 10。表示大小的占用从 8 字节变成 1 字节

翻译成 size 的指数？或者就干脆就 size_exp 或者就 大小的指数